### PR TITLE
Discounts base recipe

### PR DESCRIPTION
**Pull request by Steveb-p**
| :ticket: Issue | IBX-XXXXX |
|----------------|-----------|

### CODE REVIEW

File: ibexa/discounts/4.6/config/routes/ibexa_discounts.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ibexa.discounts.rest:
```
Comment (konradoboza) [inline ibexa/discounts/4.6/config/routes/ibexa_discounts.yaml:1]: `ibexa/discounts/4.6/config/routes/ibexa_discounts.yaml` is that correct? I don't see any indication that this package should be installed only for `commerce` tier.

File: ibexa/discounts/5.0/config/routes/ibexa_discounts.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+ibexa.discounts.rest:
```
Comment (konradoboza) [inline ibexa/discounts/5.0/config/routes/ibexa_discounts.yaml:1]: Same remark.

**Review by konradoboza — APPROVED** [review]
+1 after internal sync.

File: ibexa/discounts/4.6/config/routes/ibexa_discounts.yaml (hunk 1 above)
Comment (Steveb-p) [reply ibexa/discounts/4.6/config/routes/ibexa_discounts.yaml:1]: Recipes are unaware of access tiers. They either are applied due to a package presence, or they are not.

If you have the `ibexa/discounts` package installed, it's recipe will be applied by Symfony's Composer plugin.

**Review by ciastektk — APPROVED** [review]

File: ibexa/discounts/5.0/config/routes/ibexa_discounts.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    resource: '@IbexaDiscountsBundle/Resources/config/routing_rest.yaml'
+    prefix: '%ibexa.rest.path_prefix%'
+
+ibexa.discounts.admin:
+    resource: '@IbexaDiscountsBundle/Resources/config/routing_admin.yaml'
```
Comment (adamwojs) [inline ibexa/discounts/5.0/config/routes/ibexa_discounts.yaml:6]: Access should be limited to admin_group

**Review by adamwojs — CHANGES_REQUESTED** [review]

**Review by adamwojs — APPROVED** [review]

**Pull request merged**